packages/core/src/view/handler/SelectionHandler.ts

```diff
@@ -300,7 +300,7 @@ class SelectionHandler implements GraphPlugin {
   /**
    * Maximum number of cells for which live preview should be used.  Default is 0 which means no live preview.
    */
-  maxLivePreview = 0;
+  maxLivePreview = 50;
```

Comment:
Like above, please change the comment accordingly if the default value must change.